app/web/src/components/ImageUploader/ImageUploader.js

```diff
@@ -11,6 +11,26 @@ import Svg from 'src/components/Svg'
 const CLOUDINARY_UPLOAD_PRESET = 'CadHub_project_images'
 const CLOUDINARY_UPLOAD_URL = 'https://api.cloudinary.com/v1_1/irevdev/upload'
 
+export function ImageFallback({
+  width = 100,
+  imageId = 'CadHub/eia1kwru54g2kf02s2xx',
```

Comment:
not relevant to this PR, but I do think there must be a better way of doing a fallback/empty-state than this image.

Reply:
I hear that. I'll add to my list, maybe Cloudinary has a documented best.